vo_new/components/charts/ChartForm.tsx

```
@@ -0,0 +1,752 @@
+        
+        {!generatedChart && !generating && !generateError && (
+          <div className="flex items-center justify-center h-64 bg-muted/50 rounded-lg border-2 border-dashed">
+            <div className="text-center text-muted-foreground">
+              {editChart ? (
+                <>
+                  <p>Loading chart data...</p>
+                  <p className="text-sm">Please wait while we load your chart</p>
+                </>
+              ) : (
+                <>
+                  <p>Fill out the form and click "Generate Chart"</p>
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)